tests/probe/t14-specimen.sh

```
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Throwaway specimen for the T14 probe (issue #59). Deliberately defective so
+# CodeRabbit has something to flag and something to autofix. Delete with the
+# branch.
+
+collect_logs() {
+  DIR=$1
+  cd $DIR
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)